src/main/java/com/xpandit/plugins/xrayjenkins/Utils/BuilderUtils.java

```diff
@@ -31,6 +32,23 @@ public static boolean isSupportedJobType(Class<? extends AbstractProject> jobTyp
                 || MavenModuleSet.class.isAssignableFrom(jobType);
     }
 
+    /**
+     * Utility method to check if the endpoint supports glob expressions
```

Comment:
(Not sure where to put this comment)
This code change, it's correct, but it's not enough. When you support the Glob expression, you also have to add in the UI this option in the new supported multipart endpoints:

<img width="1051" alt="Screenshot at Oct 04 13-54-12" src="https://user-images.githubusercontent.com/11166836/66209298-f7f16080-e6ae-11e9-8a15-03cfd90b0425.png">